contrib/clojure-package/test/org/apache/clojure_mxnet/util_test.clj

```diff
@@ -70,8 +70,8 @@
                (util/option->value)))))
 
 (deftest test-keyword->snake-case
-  (is (= [:foo-bar :foo2 :bar-bar])
-      (util/keyword->snake-case [:foo_bar :foo2 :bar-bar])))
+  (is (= ["foo_bar" "foo2" "bar_bar"]
```

Comment:
I am not sure this is the test you wanted to write in the first place

Reply:
to test the base functionality. It was one of those cases where the dev generator was written initially without any tests and then I circled back and put tests in, so the tests don't make as much sense as if they would have if it was initially written with that intent.